src/extension/background-script/state.ts

```diff
@@ -59,6 +64,7 @@ const state = createState<State>((set, get) => ({
   connector: null,
   account: null,
   settings: DEFAULT_SETTINGS,
+  migrations: [],
```

Comment:
Could be an array of possible "migrations"? `typeof`... migration-constants?

Reply:
how do I do that?